src/frontend/components/login/Login.vue

```diff
@@ -3,10 +3,11 @@
     <v-layout align-center justify-center>
       <v-flex md4>
         <v-card>
-          <v-toolbar color="primary" dark flat>
-            <v-toolbar-title class="display-1">Login</v-toolbar-title>
+          <v-toolbar class="display-1-box" >
```

Comment:
A couple things here:
- [ ] `display-1-box` seems to be not needed.
- [ ] You can use the vuetify helper class `elevation-0` to get rid of the box shadow on this component (to match the designs).
- [ ] Then add a bottom border and you're good

```suggestion
          <v-toolbar class="elevation-0">
```